### DISCUSSION TITLE
How to open (Me stupid, plz help)

### DISCUSSION THREAD

**Discussion by kaeviniskal** [3kh0/ext-remover#646] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
YO, i dont know how to activate these exploits, so can someone help me

**Comment by obkduishinijf** [comment, marked as answer]
Which one? Point blank?

**Reply by Kazuma-Itsuchi** [reply]
Point blank would be useful if I could get a link that works for fortiguard.

**Reply by kaeviniskal** [reply]
yea, point blank or anything else
